src/dlgRoomProperties.cpp

```diff
@@ -93,7 +93,7 @@ void dlgRoomProperties::init(
         }
     }
     if (!lineEdit_roomSymbol->isHidden()) {
-        lineEdit_roomSymbol->setFont(mpHost->mpMap->mMapSymbolFont);
+        lineEdit_roomSymbol->setFont(QApplication::font());
```

Comment:
What the heck - try replacing the font setting for both the lineEdit AND the comboBoxt with:
```cpp
    auto mapSymbolFont = mpHost->mpMap->mMapSymbolFont;
    mapSymbolFont.setPointSize(qApp->font().pointSize());
    if (!lineEdit_roomSymbol->isHidden()) {
        lineEdit_roomSymbol->setFont(mapSymbolFont);
    }
    if (!comboBox_roomSymbol->isHidden()) {
        comboBox_roomSymbol->setFont(mapSymbolFont);
    }
```